src/components/Header/index.tsx

```diff
@@ -1,41 +1,47 @@
-import { Box, Flex, Link } from '@chakra-ui/react'
-import { motion } from 'framer-motion'
-import NextImage from 'next/image'
+import { Box, Flex, Button, Text } from '@chakra-ui/react'
 
 import { Container, HeaderNav, HeaderTop } from '@components'
 
 export const Header = (): JSX.Element => {
```

Comment:
We expect `Hero` component to be separated from `Header`. You may have forgotten to undo the changes you made in the `Header`.